ci/docker/install/ubuntu_docs.sh

```diff
@@ -21,8 +21,44 @@
 # the whole docker cache for the image
 
 set -ex
-wget http://downloads.lightbend.com/scala/2.11.8/scala-2.11.8.deb && \
-    dpkg -i scala-2.11.8.deb && rm scala-2.11.8.deb
+# Install dependencies
+echo 'Installing dependencies...'
+apt-get install -y \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    doxygen \
+    git \
+    libatlas-base-dev \
+    libjemalloc-dev \
+    liblapack-dev \
+    libopenblas-dev \
+    libopencv-dev \
+    pandoc \
+    python-numpy \
+    python-pip \
+    software-properties-common \
+    unzip \
+    wget
 
-apt-get install -y doxygen libatlas-base-dev graphviz pandoc
-pip install sphinx==1.3.5 CommonMark==0.5.4 breathe mock recommonmark pypandoc beautifulsoup4
+echo 'Installing Scala...'
+# Setup Scala
```

Comment:
Scala should already be installed

Reply:
My Scala setup was slightly different... and tested, so I think we should go with mine.

Reply:
The problem is that this script is supposed to only install things that are related to the document generation itself. Scala, for example, is installed in https://github.com/apache/incubator-mxnet/blob/master/ci/docker/Dockerfile.build.ubuntu_cpu#L31 and no additional installation should be made at a later point in time. If there's something missing here, it's best to make the change to the install/ubuntu_scala.sh file instead.

At the moment, your commands are overlapping with the scala install script and could thus cause conflicts or unexpected side effects

Reply:
`maven` is already installed, but not `sbt` and `scala`. Do you want me to add those to `ubuntu_scala.sh`?

Reply:
Wait, that's actually a good point! Where is the scala runtime actually installed? :O